src/test/java/org/broadinstitute/barclay/help/DocumentationGenerationIntegrationTest.java

```diff
@@ -42,46 +49,49 @@
         // set the doclet class
         docArgList.add("-doclet");
         docArgList.add(docletClass.getName());
+        // set the index and output extension
+        docArgList.add("-index-file-extension");
+        docArgList.add(indexFileExtension);
+        docArgList.add("-output-file-extension");
+        docArgList.add(outputFileExtension);
         return docArgList.toArray(new String[]{});
     }
 
     @DataProvider
     public Object[][] getDocGenTestParams() {
         return new Object[][] {
                 // default doclet and templates
-                {HelpDoclet.class, new File("src/main/resources/org/broadinstitute/barclay/helpTemplates/"), new File("src/test/resources/org/broadinstitute/barclay/help/expected/HelpDoclet")},
+                {HelpDoclet.class, new File("src/main/resources/org/broadinstitute/barclay/helpTemplates/"), new File("src/test/resources/org/broadinstitute/barclay/help/expected/HelpDoclet"), "html", "html"},
                 // custom doclet and templates
-                {TestDoclet.class, new File("src/test/resources/org/broadinstitute/barclay/help/templates/TestDoclet"), new File("src/test/resources/org/broadinstitute/barclay/help/expected/TestDoclet")}
+                {TestDoclet.class, new File("src/test/resources/org/broadinstitute/barclay/help/templates/TestDoclet"), new File("src/test/resources/org/broadinstitute/barclay/help/expected/TestDoclet"), "html", "html"}
```

Comment:
We're still only testing with html outputs. To have a valid test, we need to set the index output extension to something other than the default.